polyglotdb/config.py

```diff
@@ -2,16 +2,17 @@
 import logging
 import configparser
 
-CONFIG_DIR = os.path.expanduser('~/.pgdb')
+if 'CONDA_PREFIX' in os.environ:
+    CONFIG_DIR = os.path.join(os.environ['CONDA_PREFIX'], '.pgdb')
```

Comment:
I think a potentially better solution would be to define a dedicated environment variable for PGDB, instead of leveraging CONDA_PREFIX, which might not always be writable and is more opaque.  Something like `PGDB_ROOT` or `PGDB_HOME`. This is the approach that I've used for MFA: https://github.com/MontrealCorpusTools/Montreal-Forced-Aligner/blob/main/montreal_forced_aligner/config.py#L39C1-L39C31